client/service_test.go

```diff
@@ -246,7 +246,7 @@ func Test_Service_SaveEndpoint(t *testing.T) {
 			fields{
 				store: storeClientFailer{},
 			},
-			args{e: &store.Endpoint{Name: "testEndpoint", Type: blockchain.Substrate}},
+			args{e: &store.Endpoint{Name: "testEndpoint", Type: substrate.Name}},
```

Comment:
Maybe these should blockchain-agnostic or in different folders if are related to certain blockchain?

Reply:
The idea here was just to have a valid endpoint we could save for a test. None of the logic here is being changed, so maybe we should create an issue for it for the future instead of changing it here.

Reply:
Could you please create one @boxhock?

Reply:
:+1: #136